src/components/Footer.js

```diff
@@ -26,62 +28,124 @@ const socialMedias = [
 ];
 
 const { muiGridBlockContainer } = stylesBase;
-const useStyles = makeStyles({
+const useStyles = makeStyles(theme => ({
   muiGridBlockContainer,
   muiIconLogo: stylesBase.muiIconLogo,
-  lightIcon: { opacity: 0.6 },
+  subContainer: {
+    marginBottom: spacing(6),
+  },
+  menuContainer: {
+    [theme.breakpoints.down('xs')]: {
+      margin: 'auto',
+    },
+    [theme.breakpoints.down('sm')]: {
+      marginBottom: spacing(5),
+    },
+  },
+  lightIcon: ({ last }) => ({
+    opacity: 0.6,
+    marginRight: last ? '0px' : spacing(3),
+  }),
   itemContainer: {
     width: 'auto',
+    [theme.breakpoints.down('xs')]: {
+      marginBottom: spacing(4),
+    },
+  },
+  itemTextContainer: {
+    marginBottom: spacing(4),
   },
   itemText: {
     opacity: 1,
     fontWeight: 'bold',
-    marginBottom: spacing(4),
   },
   links: { textDecoration: 'none', marginBottom: spacing(4) },
-});
+}));
 
-const Footer = () => {
-  // eslint-disable-next-line no-shadow
-  const { muiGridBlockContainer, muiIconLogo, lightIcon, itemContainer, itemText, links } = useStyles();
+const Footer = ({ width }) => {
+  const {
+    // eslint-disable-next-line no-shadow
+    muiGridBlockContainer,
+    muiIconLogo,
+    subContainer,
+    menuContainer,
+    itemContainer,
+    itemTextContainer,
+    itemText,
+    links,
+  } = useStyles();
   return (
-    <Grid className={muiGridBlockContainer} container justify="flex-end">
-      <Grid item container xs={6} direction="column">
+    <Grid
+      className={muiGridBlockContainer}
+      container
+      justify="flex-end"
+      wrap={isWidthDown('sm', width) ? 'wrap-reverse' : 'wrap'}
+    >
+      <Grid
+        className={subContainer}
+        item
+        container
+        direction="column"
+        xs={isWidthDown('sm', width) ? 12 : 5}
```

Comment:
This isn't the right way to use the [column breakpoints with `Grid`](https://material-ui.com/components/grid/#grid-with-breakpoints). You can specify multiple values for `xs`, `sm`, etc. to automatically be responsive for difference sizes. In this case, you'd want `<Grid xs={12} md={5} ...>`